www-client/casperjs/casperjs-1.1_beta3.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v3
+# $Id$
+
+EAPI=5
+
+MY_PV=${PV/_beta/-beta}
+
+DESCRIPTION="Navigation scripting & testing utility for PhantomJS and SlimerJS"
+HOMEPAGE="http://casperjs.org/"
+SRC_URI="https://github.com/n1k0/${PN}/archive/${MY_PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${PN}-${MY_PV}"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+DEPEND="<www-client/phantomjs-2.0.0"
+RDEPEND="${DEPEND}"
+
+src_compile() {
+	return
+}
+
+src_install() {
+	dodir /usr/share/${P}/
+	insinto /usr/share/${P}/
+	doins -r modules/
+	doins -r tests/
+	doins package.json
+
+	exeinto /usr/share/${P}/bin
```

Comment:
This one may be problematic without a big patch since the files in `./bin` locate other files by a relative file path (c.f. https://github.com/n1k0/casperjs/blob/master/bin/casperjs#L130, https://github.com/n1k0/casperjs/blob/master/bin/bootstrap.js#L175, https://github.com/n1k0/casperjs/blob/master/bin/bootstrap.js#L380)